HLTrigger/Configuration/python/HLT_75e33/eventsetup/hltOnlineBeamSpotESProducer_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
 hltOnlineBeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer",
-    timeThreshold = cms.int32( 48 ),
+    timeThreshold = cms.int32( 1e6 ),   # we do want to read the DB even if it's old
```

Comment:
mmh, 
> ValueError: 1000000.0 is not a valid <class 'FWCore.ParameterSet.Types.int32'>

Reply:
maybe try `999999`? as in:
https://github.com/cms-sw/cmssw/blob/a52f2d1ecf97bde0e9f10d45959c88337c481687/Configuration/DataProcessing/python/RecoTLR.py#LL10C25-L10C31

Reply:
the HLT customization has `int(1e6)`:

https://github.com/cms-sw/cmssw/blob/fa01ae41a33c9ecaa9771e2259c455a6e942f068/HLTrigger/Configuration/python/customizeHLTforCMSSW.py#LL209C30-L210C1

I overlooked it yesterday.